db/fetch_db.rb

```diff
@@ -0,0 +1,87 @@
+require_relative '../lib/poll'
+require_relative '../lib/vote'
+require 'pg'
+
+=begin
+  データベースにアクセスして投票や票の
+  情報を取得する、また票のカウントを
+  行うクラス
+=end
+
+class Fetch_DB
```

Comment:
> Rubyのクラスとメソッドの定義の仕方

Ruby のクラス名はアッパーキャメルケースが慣習なので、アンダーバーは取り除いてください。
また、クラス名は名詞にしたいので個人的には `DBFetcher` とかの方が好みです。
さらに、Fetcher という名前だと読み込みしかしなそうですが、`#vote` では書き込みも行うので誤解を生みそうです。
この名前がいいか分からないですが `DBWrapper` とかそのまま `Database` とかでもいいのかなと思います。

```suggestion
class DBWrapper
```